.custom_wordlist.txt

```diff
@@ -240,11 +242,13 @@ runtimes
 rustc
 rustup
 RSA
+SaaS
 scalability
 SDK
 SeaBIOS
 SEV
 SHA
+signup
```

Comment:
is there a better way to do this? the flag seems to be on a link which i wouldn't expect to be spell checked.